Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -377,8 +374,9 @@
 					};
 					E4BF2DEB1BACE92400DA9D68 = {
 						CreatedOnToolsVersion = 7.0;
-						DevelopmentTeam = 43AQ936H96;
+						DevelopmentTeam = 9G8J6YA743;
 						LastSwiftMigration = 0800;
+						ProvisioningStyle = Manual;
```

Comment:
This is most of the change. I was running into issues with Xcode's automatic code signing management because I wanted to be able to build a target using a different team based on which scheme we're running (Focus -> Company, Focus (Enterprise) -> Enterprise). The only way I could get it to work is if I turned on manual code signing management and generated the profiles in Xcode myself.